bundles/org.openhab.binding.teleinfo/src/main/resources/ESH-INF/thing/serialController.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<bridge-type id="serialcontroller">
+		<label>Teleinfo Serial Controller</label>
+		<description>Teleinfo USB Stick with Serial Interface</description>
+
+		<channels>
+			<channel id="invalidFrameCounter" typeId="invalidFrameCounterType"/>
+		</channels>
+
+		<config-description>
+			<parameter name="serialport" type="text" required="true">
+				<label>Serial Port</label>
+				<context>serial-port</context>
+				<description>Serial port of Teleinfo device (e.g.: /dev/ttyUSB0 on Linux, COM1 on Windows)</description>
+				<limitToOptions>false</limitToOptions>
+			</parameter>
+			<parameter name="autoRepairInvalidADPSgroupLine" type="boolean" required="false">
+				<label>Auto repair malformed ADPS data</label>
```

Comment:
Words in labels should be capitalized. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions